backend/api/Controllers/RobotController.cs

```diff
@@ -573,6 +573,69 @@ public async Task<ActionResult> ResumeMission([FromRoute] string robotId)
         return NoContent();
     }
 
+
+    /// <summary>
+    /// Post new armposition (Battery pose, Transport pose, Lookout pose) for the robot with id 'robotId'
+    /// </summary>
+    /// <remarks>
+    /// <para> This query moves the arm to a given position for a given robot </para>
+    /// </remarks>
+    [HttpPut]
+    [Authorize(Roles = Role.User)]
+    [Route("{robotId}/SetArmPosition/{armPosition}")]
+    [ProducesResponseType(typeof(Mission), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult> SetArmPosition(
+        [FromRoute] string robotId,
+        [FromRoute] string armPosition
+    )
+    {
+        var robot = await _robotService.ReadById(robotId);
+        if (robot == null)
+        {
+            _logger.LogWarning("Could not find robot with id={id}", robotId);
+            return NotFound("Robot not found");
+        }
+
+        if (robot.Status is not RobotStatus.Available)
+        {
+            _logger.LogWarning(
+                "Robot '{id}' is not available ({status})",
+                robotId,
+                robot.Status.ToString()
+            );
+            return Conflict($"The Robot is not available ({robot.Status})");
+        }
+        try
+        {
+            await _isarService.StartMoveArm(robot, armPosition);
+        }
+        catch (HttpRequestException e)
+        {
+            string message = "Error connecting to ISAR while resuming mission";
+            _logger.LogError(e, "{message}", message);
+            OnIsarUnavailable(robot);
+            return StatusCode(StatusCodes.Status502BadGateway, message);
+        }
+        catch (MissionException e)
+        {
+            _logger.LogError(e, "Error while resuming ISAR mission");
```

Comment:
This message should probably reflect what went wrong

Reply:
done